lib/server.js

```diff
@@ -55,7 +55,9 @@ function createServer (options, httpHandler) {
     } else {
       host = listenOptions.host
     }
-    if (Object.prototype.hasOwnProperty.call(listenOptions, 'host') === false) {
+    if (Object.prototype.hasOwnProperty.call(listenOptions, 'host') === false ||
+      listenOptions.host === undefined ||
+      listenOptions.host === null) {
```

Comment:
This will have the same behavior

```suggestion
    if (listenOptions.host == null) {
```

Reply:
fixed! thanks @gurgunday